aicsimageio/tests/image_container_test_utils.py

```diff
@@ -143,7 +145,9 @@ def run_image_file_checks(
     expected_dtype: np.dtype,
     expected_dims_order: str,
     expected_channel_names: Optional[List[str]],
-    expected_physical_pixel_sizes: Tuple[float, float, float],
+    expected_physical_pixel_sizes: Tuple[
+        Optional[float], Optional[float], Optional[float]
```

Comment:
minor nitpick but can you have only one of them be None but not the others?  Do we really mean Tuple[float, float, float] | Tuple[None,None,None] ?

Reply:
Ah, I see in the tests you have Z=None sometimes...